lib/fizzy/parser_expr.cpp

```diff
@@ -76,6 +91,23 @@ void update_caller_frame(ControlFrame& frame, const FuncType& func_type)
     frame.stack_height += stack_height_change;
 }
 
+void validate_result_count(const ControlFrame& frame)
+{
+    if (!frame.unreachable)
+    {
+        // This is checked by "stack underflow".
+        assert(frame.stack_height >= frame.parent_stack_height);
+
+        if (frame.stack_height - frame.arity < frame.parent_stack_height)
```

Comment:
just a minor nitpick, but to me it would look more straightforward as
```suggestion
        if (frame.stack_height < frame.parent_stack_height + frame.arity)
```
or
```suggestion
        if (frame.stack_height - frame.parent_stack_height < frame.arity)
```
first one preferable

Reply:
Make sense, I will use `frame.parent_stack_height + frame.arity`.